riscv/csrs.cc

```diff
@@ -1,5 +1,7 @@
 // See LICENSE for license details.
 
+#include <algorithm>
```

Comment:
Please add a comment just above: "For std::any_of:" so we know why this is here

Reply:
Done